### PR TITLE
add copyright linter

### PR DESCRIPTION
**Pull request by aruniverse**
Add copyright linter

### CODE REVIEW

File: common/scripts/package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@bentley/viewer-components-react-scripts",
```
Comment (calebmshafer) [inline common/scripts/package.json:2]: So I don't think you need a package.json.. I was testing this out in the iModel.js repo and pretty sure I figured out a way to get around it.  At least for the copyright header case.

Comment (calebmshafer) [reply common/scripts/package.json:2]: Let me dig up that branch, I didn't get around to moving forward with it yet but would like to soon.

Comment (aruniverse) [reply common/scripts/package.json:2]: If we can get around that, thatd be great. I just copied what the viewer was doing. Just need to make sure lint-staged is installed.

Comment (aruniverse) [reply common/scripts/package.json]: Updated, removed the unnecessary pkg, and tested it out locally. This should be added to all the other imjs repos as well

**Pull request merged**